Tests/SonarRunner.Shim.Tests/PropertiesWriterTest.cs

```diff
@@ -95,11 +92,11 @@ public void PropertiesWriterToString()
 {0}\\\u4F60\u597D.cs,\
 {4}
 
-DB2E5521-3172-47B9-BA50-864F12E6DFFF.sonar.projectKey=my_project_key:DB2E5521-3172-47B9-BA50-864F12E6DFFF
-DB2E5521-3172-47B9-BA50-864F12E6DFFF.sonar.projectName=\u4F60\u597D
-DB2E5521-3172-47B9-BA50-864F12E6DFFF.sonar.projectBaseDir={0}
-DB2E5521-3172-47B9-BA50-864F12E6DFFF.sonar.vbnet.fxcop.reportPath={1}
-DB2E5521-3172-47B9-BA50-864F12E6DFFF.sonar.sources=\
```

Comment:
I updated the guids used in the test to make them unique (two of the projects were using the same guid).